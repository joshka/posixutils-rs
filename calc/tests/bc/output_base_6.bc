obase = 6
0
1
1034
238.672
quit
