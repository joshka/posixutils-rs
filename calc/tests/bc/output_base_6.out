0
1
4442
1034.4010
